.circleci/config.yml

```diff
@@ -1896,108 +1728,6 @@ jobs:
-          filters: *filter-only-master
-      - codespell:
-          filters: *filter-only-master
-      - lint-go:
-          filters: *filter-only-master
-      - shellcheck:
-          filters: *filter-only-master
-      - test-frontend:
-          filters: *filter-only-master
-      - test-backend:
-          filters: *filter-only-master
-      - mysql-integration-test:
-          filters: *filter-only-master
-      - postgres-integration-test:
-          filters: *filter-only-master
-      - deploy-master:
-          requires:
-            - build-all
-            - test-backend
-            - test-frontend
-            - codespell
-            - lint-go
-            - shellcheck
-            - mysql-integration-test
-            - postgres-integration-test
-            - build-oss-msi
-          filters: *filter-only-master
-      - grafana-docker-master:
-          requires:
-            - build-all
-            - build-all-enterprise
-            - test-backend
-            - test-frontend
-            - codespell
-            - lint-go
-            - shellcheck
-            - mysql-integration-test
-            - postgres-integration-test
-          filters: *filter-only-master
-      - grafana-docker-ubuntu-master:
-          requires:
-            - build-all
-            - build-all-enterprise
-            - test-backend
-            - test-frontend
-            - codespell
-            - lint-go
-            - shellcheck
-            - mysql-integration-test
-            - postgres-integration-test
-          filters: *filter-only-master
-      - deploy-enterprise-master:
-          requires:
-            - build-all
-            - test-backend
-            - test-frontend
-            - codespell
-            - lint-go
-            - shellcheck
-            - mysql-integration-test
-            - postgres-integration-test
-            - build-all-enterprise
-            - build-ee-msi
-          filters: *filter-only-master
-      - build-oss-msi:
-          requires:
-            - build-all
-            - test-backend
-            - test-frontend
-            - codespell
-            - lint-go
-            - shellcheck
-            - mysql-integration-test
-            - postgres-integration-test
-          filters: *filter-only-master
-      - build-ee-msi:
-          requires:
-            - build-all-enterprise
-            - test-backend
-            - test-frontend
-            - codespell
-            - lint-go
-            - shellcheck
-            - mysql-integration-test
-            - postgres-integration-test
-          filters: *filter-only-master
-      - end-to-end-test:
-          requires:
-            - grafana-docker-master
-          filters: *filter-only-master
-      - release-next-packages:
-          requires:
-            - build-all
-            - test-frontend
-          filters: *filter-only-master
-
-  # Pipeline for building master branch, based on Grafana Build Pipeline tool
   build-pipeline:
```

Comment:
Nit, seems like this is only used for master so maybe change the name to denote master?

Reply:
@marefr Good observation, but my next step is to try to apply the same workflow for PR and release pipelines also (through branching logic) :)

Reply:
Ohh nice makes perfect sense. Please disregard this then for now.

Reply:
OK :) I don't know for sure yet if I can use a single workflow as I hope, but will try.